### DISCUSSION TITLE
How to configure "Edit with Emacs" with `use-package`?

### DISCUSSION THREAD

**Discussion by logc** [agzam/spacehammer#185] [Q&A]
I have never gotten around to making "Edit with Emacs" work on my setup. 

I am trying to add `spacehammer.el` to my configuration, using `use-package`. This did not work out of the box because `use-package` tries to find it on the remote repositories, however much I try to declare that it is already present locally. 

Can someone please show me an example of a working config? Thank you very much!

**Comment by agzam** [comment]
As stated in the docs, you don't have to load the package from the local source, you can set the recipe to load the package from GitHub. https://github.com/agzam/spacehammer/blob/master/docs/emacs.org#load-the-package-from-github
You may have to use quelpa-use-package or if you're using a package manager, you can do it directly. see for example: https://github.com/jwiegley/use-package/issues/410
